src/css/components/nav.scss

```diff
@@ -37,16 +37,24 @@
     &:hover {
       color: $color-primary;
     }
+    &[href *= ".io"] {
```

Comment:
Do you expect that this icon will only be used in the context of an element with a `nav-link` class? If not, we might try putting this icon somewhere more generic.